pkgs/applications/misc/qolibri/default.nix

```
@@ -3,13 +3,13 @@
 
 stdenv.mkDerivation rec {
```

```suggestion
mkDerivation rec {
```

And the application will automatically be wrapped (currently fails at runtime).
Preferably in a separate commit.

---

Thanks. I didn't see a failure at runtime for some reason, but I have updated it to use `mkDerivation`.